docs/spec/Expectations.md

```diff
@@ -0,0 +1,77 @@
+# Expectations
+## GET /expectations/
+Returns an enumeration of all expectations, along with the touchstone version id, modelling group and disease to which
+they apply. Expectations are returned only for open touchstones and responsibilities.
+
+Required permissions: */responsibilities.read
+
+Schema: [`TouchstoneModelExpectations.schema.json`](../schemas/TouchstoneModelExpectations.schema.json)
+
+#### Example
+
+    [
+        {
+            "touchstone_version": "2017-op-1",
+            "modelling_group": "IC-Garkse",
+            "disease": "YF",
+            "expectations": {
+                "id": 12,
+                "description": "Expectations for pilot run",
+                "years": {
+                    "minimum_inclusive": 2000,
+                    "maximum_inclusive": 2100
+                },
+                "ages": {
+                    "minimum_inclusive": 0,
+                    "maximum_inclusive": 99
+                },
+                "cohorts": {
+                    "minimum_birth_year": 1950,
+                    "maximum_birth_year": 2050
+                },
+                "countries": [
+                    {
+                        "id": "AFG",
+                        "name": "Afghanistan"
+                    },
+                    {
+                        "id": "AGO",
+                        "name": "Angola"
+                    }
+                ],
+                "outcomes": ["deaths", "cases"]
+            }
+        },
+        {
+            "touchstone_version": "2018-op-1",
+            "modelling_group": "IC-Garkse",
+            "disease": "YF",
+            "expectations": {
+                "id": 13,
+                "description": "Expectations for pilot run",
+                "years": {
+                    "minimum_inclusive": 2000,
+                    "maximum_inclusive": 2100
+                },
+                "ages": {
+                    "minimum_inclusive": 0,
+                    "maximum_inclusive": 99
+                },
+                "cohorts": {
+                    "minimum_birth_year": 1960,
+                    "maximum_birth_year": 2060
+                },
+                "countries": [
```

Comment:
Given this implementation countries and outcomes will always be empty. We could exclude countries if we want, since not needed for the admin portal work, but in that case we shouldn't return the field at all (an empty array will be misleading!)